python/res/job_queue/job_manager.py

```diff
@@ -293,8 +293,7 @@ def startStatus(self, job):
             now = time.localtime()
             f.write("%-32s: %02d:%02d:%02d .... " % (job["name"], now.tm_hour, now.tm_min, now.tm_sec))
 
-
-    def completeStatus(self, exit_status, error_msg, job=None):
+    def markJobComplete(self, job, exit_status, error_msg):
```

Comment:
Should we make `error_msg` default to the empty string? And then possibly raise an exception if the `exit_status` is not `0`?

Reply:
Or even default to `None` perhaps?

Reply:
Sat default `error_msg` to `None`, but non-zero `exit_status` is perfectly fine for completed jobs.